src/main/java/org/jabref/model/entry/FileFieldParser.java

```diff
@@ -1,59 +1,58 @@
 package org.jabref.model.entry;
 
 import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 
 public class FileFieldParser {
-    public static List<LinkedFile> parse(String value) {
-        if ((value == null) || value.trim().isEmpty()) {
-            return Collections.emptyList();
-        }
 
+    public static List<LinkedFile> parse(String value) {
```

Comment:
Returning a mutable list here is ok, since the method is static anyway and thus you don't have the problem that a mutable list exposes the internal state of the object to the outside world.
However, `getFiles` is not static and only used once...

As I said above, I have no well-founded knowledge about these matters but probably there a established conventions that we should follow here.